action.yaml

```diff
@@ -25,6 +25,9 @@ inputs:
     description: Name of function file inside function folder, for instance, handler.py.
     default: handler.py
     required: false
+  function_secrets:
+    description: Base64 encoded string with json payload. Will be passed to a function's secrets as is.
+    required: false
```

Comment:
As is? It sounds like FunctionsAPI.create kwarg `secrets` accepts this

Reply:
yup. that's the case. we just put that dictionary there and it will be given to the function implementation as it was stored here. (already decoded and parsed into dict)